tests/test_roles.py

```diff
@@ -1,12 +1,20 @@
 from http import HTTPStatus
 
+from flask import url_for
+
+from timeless.roles.models import Role
+
 
 def test_list(client):
     assert client.get("/roles/").status_code == HTTPStatus.OK
 
 
 def test_create(client):
-    assert client.get("/roles/create").status_code == HTTPStatus.OK
+    response = client.post(url_for("role.create"), data={
+        "name": "owner"
+    })
+    assert response.location.endswith(url_for('role.list_roles'))
+    assert Role.query.count() == 1
```

Comment:
@Vatavuk May be let's check exactly this row, not checking only count? WDYT?

Reply:
@vladarefiev 👍